app/models/knight.rb

```diff
@@ -1,2 +1,17 @@
 class Knight < ChessPiece
+  XY_MOVE_FACTOR = 1
+  ALT_XY_MOVE_FACTOR = 2
+
+  def valid_move?(destination_x, destination_y)
+    return false if out_of_bounds?(destination_x, destination_y)
+    return false unless x_and_y_are_offset?(destination_x, destination_y)
+    true
+  end
+
+  private
+
+  def x_and_y_are_offset?(destination_x, destination_y)
+    ((destination_x - position_x == XY_MOVE_FACTOR) && (destination_y - position_y == ALT_XY_MOVE_FACTOR)) ||
+      ((destination_y - position_y == XY_MOVE_FACTOR) && (destination_x - position_x == ALT_XY_MOVE_FACTOR))
```

Comment:
I think you need to get the absolute value of the difference here(like you did in king).
I think this test will fail.
```ruby
it 'returns true if the x and y move components are -1 and 2, respectively' do
 expect(knight.valid_move?(3, 6)).to eq true
end
```